### ISSUE TITLE
Remove the /stats module

### ISSUE THREAD

**Issue by rjsparks** [ietf-tools/datatracker#3712]
The /stats module as it currently exists is an attractive nuisance. It was intended to move the nice things that Jari has been maintaining into the datatracker, but the implementation didn't really achieve that.

As it is, what's presented ranges from incomplete to actually wrong.

We're going to remove it and schedule a new effort to build better statistics.

**Comment by rjsparks** [comment]
There are a few things, like the MeetingRegistration model, that should have been in other apps to begin with that will have to be moved.

**Comment by rjsparks** [comment]
Tero mentioned at <https://mailarchive.ietf.org/arch/msg/tools-discuss/0Tw_KT4pYdUxoIf36p8yzn_bvEM> that some parts of the existing implementation may remain useful, and we should review whether/what to move into the relevant apps while waiting for re-implementation.

**Comment by larseggert** [comment]
Are you still interested in a PR that rips out the document stats?

**Comment by rjsparks** [comment]
Yes, and the meeting stats. It would be best if the same PR moved the review-team stats into the review application.

**Comment by jennifer-richards** [comment]
The `stats` app still exists, but has been mostly removed. Closing this. The work to replace it is tracked as #8511
